tests/foreman/api/test_virt_who_config.py

```diff
@@ -71,13 +78,16 @@ def test_postive_config_intervals(self):
 
         :steps:
             1. Create virt-who configuration, set reporting interval to 1 hour.
```

Comment:
set reporting interval - is this something needs to be set under settings ? or under config file ?

Reply:
This is a field when creating a virt-who configuration.  will change this to 
```
Create virt-who configuration with reporting interval to 1 hour.
```
To make it clearer